todo-list-ui/.vscode/extensions.json

```diff
@@ -0,0 +1,4 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=827846
+  "recommendations": ["angular.ng-template"]
```

Comment:
Let's add this extension here: `"editorconfig.editorconfig"`

That's the editorconfig vscode extension, please install it in your local too, it'll help comply the `.editorconfig` spec